hw2/hw2_1/main.js

```diff
@@ -0,0 +1,129 @@
+const express = require('express');
+const fs = require('fs');
+
+const app = express();
+const jsonParser = express.json();
+
+const port = 8080;
+
+app.use(express.static('public'));
+
+const filePath = 'hw2/hw2_1/users.json';
+
+app.get('/api/users', (req, res) => {
+  if (!req.query.name || !req.query.limit) return res.status(404).end('Oops... We need parameters NAME and LIMIT');
+
+  const contentJSON = fs.readFileSync(filePath, 'utf8');
+  const users = JSON.parse(contentJSON);
+  const resultArr = [];
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const key in users) {
```

Comment:
Here it's better to iterate through an object using ```Object.keys``` and than filter results using ```Array.prototype.filter```